src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_Window.cs

```diff
@@ -17,6 +17,16 @@ public void When_CreateNewWindow()
 			// This used to crash on wasm which was trying to create a second D&D extension
 			var sut = new Window();
 		}
+#endif
+#if __SKIA__
+		[TestMethod]
+		[RunsOnUIThread]
+		public void Verify_Window_Title_Content()
+		{
+			string SUT = Windows.UI.ViewManagement.ApplicationView.GetForCurrentView().Title;
+			string value = Windows.ApplicationModel.Package.Current.DisplayName;
+			Assert.IsTrue(SUT == value);
```

Comment:
You can use `Assert.AreEqual`, which shows both values when the assertion is false.